### PR TITLE
[pdex] Ensure not out of range with case uint64()

### PR DESCRIPTION
**Pull request by tin-incognito**
Ensure not out of range with case uint64()

### CODE REVIEW

**Review by cuong-incognito — APPROVED** [review]

File: blockchain/pdex/utils.go [hunk 1]
```diff
@@ -814,6 +814,9 @@ func getWeightedFee(txs []metadata.Transaction, pairs map[string]*PoolPairState,
 			// convert the fee from PRV to equivalent token by applying discount percent
 			temp.Mul(temp, big.NewInt(100))
 			temp.Div(temp, discountPercent)
+			if !temp.IsUint64() {
```
Comment (dat-incognito-org) [inline blockchain/pdex/utils.go:817]: should add to `invalid transactions` like below ?

File: blockchain/pdex/utils.go [hunk 2]
```diff
@@ -838,6 +841,9 @@ func getWeightedFee(txs []metadata.Transaction, pairs map[string]*PoolPairState,
 					invalidTransactions = append(invalidTransactions, tx)
 					continue
 				}
+				if !temp.IsUint64() {
```
Comment (dat-incognito-org) [inline blockchain/pdex/utils.go:844]: this is checked just above

File: blockchain/pdex/utils.go (hunk 1 above)
Comment (tin-incognito) [reply blockchain/pdex/utils.go:817]: fixed!

File: blockchain/pdex/utils.go (hunk 2 above)
Comment (tin-incognito) [reply blockchain/pdex/utils.go]: fixed!

**Review by dat-incognito-org — APPROVED** [review]

**Pull request merged**